tsconfig.json

```diff
@@ -4,10 +4,12 @@
     "baseUrl": "./src/",
     "declaration": true,
     "esModuleInterop": true,
-    "lib": ["dom", "es2020"],
+    "lib": ["DOM", "ES2020"],
     "noImplicitAny": true,
     "outDir": "./dist",
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "target": "ES2020",
+    "moduleResolution": "node"
```

Comment:
nit but mind alphabetizing this (i.e., moving the `moduleResolution` field after `lib`)?